core/src/internal/image_utils.cpp

```
@@ -40,8 +40,9 @@ namespace img2num {
 // image: pointer to RGBA data
 // width, height: dimensions
 // sigma: standard deviation of Gaussian blur
-void gaussian_blur_fft(uint8_t *image, size_t width, size_t height, double sigma_pixels) {
-    if (!image || width == 0 || height == 0 || sigma_pixels <= 0) return;
+void gaussian_blur_fft(uint8_t* image, size_t width, size_t height, double sigma_pixels) {
```

Cppcheck (2.20.0)

[style] The function 'gaussian_blur_fft' is never used.

(unusedFunction)